tests/sars_cov_2/test_generate_pipeline_results_files.py

```diff
@@ -181,7 +177,11 @@ def test_generate_pipeline_results_files(
         calc_resultfiles_json_dict = json.load(json_fd)
 
     exp_result_files_json_dict_full_path = {
-        sample_id: [f"{str(tmp_path)}/{f}" for f in files] for sample_id, files in exp_resultfiles_json_dict.items()
+        sample_id: sorted([f"{str(tmp_path)}/{f}" for f in files])
+        for sample_id, files in exp_resultfiles_json_dict.items()
+    }
+    calc_result_files_json_dict_sorted = {
+        sample_id: sorted(files) for sample_id, files in calc_resultfiles_json_dict.items()
     }
 
-    assert exp_result_files_json_dict_full_path, calc_resultfiles_json_dict
```

Comment:
This was a bug in the unit test, which did not fail the test following the recent changes.